projects/saturn/application.yaml

```diff
@@ -42,6 +42,8 @@ auth:
   clientId: workspace-client
   enableBasicAuth: true
   superAdminUser: organisation-admin
+  defaultRoles:
+    - canViewPublicData
```

Comment:
I wouldn't make this one as default. For fns we need `canViewPublicMetadata` instead, but maybe anyway it's more safe to have it empty by default and have it configurable by helm in configmap-saturn.yaml to set it only for fns deployment

Reply:
for now we can have it canViewPublicMetadata , maybe change with Synergy if not needed. I guess most customers want view public metadata on by default, it's public :)

Reply:
yes that was a mistake, good you spot it :)